src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
```

Comment:
The example with something looking like h2 but not being h2 was an example for a rare exception to my argument that creating semantic markup paves the way to do SEO easily, aside that it is not relevant and I don't like that practice either.